DataDefinitions/FactionReport.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Newtonsoft.Json;
+using System;
+
+namespace EddiDataDefinitions
+{
+    public class FactionReport
```

Comment:
This is a nitpick, but I find the name `FactionReport` to not be very intuitive as far as understanding what this does and how it differs from a `Faction` object. Consider `CrimeTransaction` as an alternate?